web/src/styles/onboarding.css

```
@@ -2426,31 +2692,98 @@
-  height: 16px;
-  accent-color: var(--accent);
+  /* Native checkbox kept in DOM for accessibility (Space toggles,
+     screen readers announce state) but visually hidden — the chip's
+     border/background + tick icon are the visible affordance. */
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)